.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,51 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

Comment:
want to scope this to only run on tags starting with `v`

Reply:
Added to file